qa/tasks/vstart_runner.py

```diff
@@ -506,6 +509,8 @@ def __init__(self):
         # certain teuthology tests want to run tasks in parallel
         self.lock = threading.RLock()
 
+        self.log = lambda x: log.info(x)
```

Comment:
I must not be interpreting this syntax properly. I'd expect to alias ```log("foo")``` to ```log.info("foo")``` but I don't even see it being referenced anywhere. What's happening?